ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -724,7 +724,7 @@ private[testtool] final class ParticipantTestContext private[participant] (
   def prune(
       pruneUpTo: LedgerOffset,
       attempts: Int = 10,
-      pruneAllDivulgedContracts: Boolean = true,
+      pruneAllDivulgedContracts: Boolean = false,
```

Comment:
There are only three test cases in `ParticipantPruningIT` that specifically test pruning divulged contracts. For these cases we enable `pruneAllDivulgedContracts` explicitly, leaving the flag disabled for all other cases.